kolibri/auth/migrations/0004_auto_20170816_1607.py

```diff
@@ -13,13 +14,19 @@ def device_owner_to_super_user(apps, schema_editor):
     default_facility = Facility.objects.all().first()
     DevicePermissions = apps.get_model('device', 'DevicePermissions')
     DeviceSettings = apps.get_model('device', 'DeviceSettings')
-    from kolibri.auth.models import FacilityUser as RealFacilityUser, Facility as RealFacility
+    Role = apps.get_model('kolibriauth', 'Role')
+    from kolibri.auth.models import FacilityUser as RealFacilityUser, Facility as RealFacility, Role as RealRole
     real_default_facility = RealFacility.get_default_facility()
     # Can't do much if no facilities exist, as no facility to FK the users onto
     if default_facility:
         for device_owner in DeviceOwner.objects.all():
-            uuid = RealFacilityUser(username=device_owner.username, facility=real_default_facility).calculate_uuid()
             dataset_id = real_default_facility.dataset_id
+            real_superuser = RealFacilityUser(
```

Comment:
are attributes defined on the models at this point?
https://github.com/learningequality/kolibri/blob/develop/kolibri/auth/models.py#L458
We need this to correctly calculate the ID.